__tests__/app.test.js

```diff
@@ -34,3 +34,48 @@ describe("app /api/topics returns 200", () => {
     });
   });
 });
+
+describe("app /api/ returns 200", () => {
+    describe("/api/", () => {
+      describe("GET /api/ returns 200", () => {
+        test("status code: 200", () => {
+          return request(app).get("/api/topics").expect(200);
+        });
+      });
+  
+      test("Will return object, containing slug and description properties", () => {
+        return request(app).get("/api/").expect(200).then(({body}) => {
+          expect(body).toEqual({
```

Comment:
Instead of hardcoding the whole content here and checking if they are the same, please require in the file contents in the test file and compare it to that one, as if you extend the json file later, this test will fail.

Reply:
Thanks Bela, just made these changes and made a new PR